proto/tikvpb.proto

```diff
@@ -54,4 +54,74 @@ service Tikv {
     // transaction debugger commands.
     rpc MvccGetByKey(kvrpcpb.MvccGetByKeyRequest) returns (kvrpcpb.MvccGetByKeyResponse) {}
     rpc MvccGetByStartTs(kvrpcpb.MvccGetByStartTsRequest) returns (kvrpcpb.MvccGetByStartTsResponse) {}
+
+    rpc SuperBatch(stream SuperBatchRequest) returns (stream SuperBatchResponse) {}
+}
+
+message SuperBatchRequest {
+    repeated Request requests = 1;
+    repeated uint64 request_ids = 2;
```

Comment:
Maybe better to add an id field in `Request` and `Response`.

Reply:
Really? I think seperating them can make the change minimum.

Reply:
Each request has an ID is more reasonable in structure. Otherwise you need more work to match the requests and their IDs, such as dealing with inconsistent lengths.

Reply:
I tried add the `id` field into `Response`. There are many things to change in `TiKV`. And because in protobuf 3 if we don't set the field to a value, it will have a default value `0`. It's more hard to find problems than seperating them from `requests`. So, let's keep it!